code/datums/flock/flock.dm

```diff
@@ -372,8 +374,10 @@
 	// update annotations indicating enemies for flockmind and co
 	src.updateAnnotations()
 
-/datum/flock/proc/removeEnemy(var/mob/living/M)
+/datum/flock/proc/removeEnemy(var/M)
```

Comment:
Why does `proc/updateEnemy` uses `var/atom/M` over `var/M`?

Reply:
atom has .setloc() and a bunch of other defines that I needed at some point in the code, and so for consistency, all references to enemies were made into atoms. This should probably also be var/atom/M